test/tools/project-cleaner/resource_spec.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project-id: ml-pipeline-test
+resources:
+  - resource: gke-cluster
+    zones:
+      - us-east1-b
+    name-prefix: e2e_
```

Comment:
as discussed offline, consider e2e- and sample-. 
A better approach might be tagging the kubernetes with special labels when creating them. And delete the cluster with those labels, instead of using name prefix as identifier.

Reply:
As per offline discussion, will be moving the tag based cluster creation and deletion to the next iteration. I have updated the yaml to handle "e2e-" and "sample-"